client/src/routes.js

```diff
@@ -0,0 +1,40 @@
+const baseUrl = "http://localhost:9000";
+const apiUrl = `${baseUrl}/api`;
+const authApiUrl = `${apiUrl}/auth`;
+const ingestApiUrl = `${apiUrl}/ingest`;
+const loginUrl = `${authApiUrl}/login`;
+const logoutUrl = `${authApiUrl}/logout`;
+const testLoginAsChris = `${authApiUrl}/user/confirm/V1StGXR8_Z5jdHi6B-myT`;
```

Comment:
Since `testLoginAsChris` is a temporary hack for testing purposes, would it make more sense to definite locally where it is used rather than putting it here?